src/constants/index.tsx

```diff
@@ -21,7 +21,7 @@ import { ReactNode } from 'react'
 import RightArrow from '../assets/images/arrow-right.svg'
 import BaoswapLogo from '../assets/images/baoswap-logo.png'
 import Coinbase from '../assets/images/coinbase.svg'
-import CurveLogo from '../assets/images/curve-logo.svg'
+import CurveLogo from '../assets/images/curve-logo.png'
 import DFYNLogo from '../assets/images/dfyn-logo.svg'
 import LevinswapLogo from '../assets/images/levinswap-logo.svg'
 import ZeroXLogo from '../assets/images/logos/ZeroX.svg'
```

Comment:
Couldn't we compress the curve logo svg?

Reply:
I tried that first. But the curve logo is a huge SVG with so many gradient effects. A compressed version itself is very big and we use only a 32*32 size of it in swapr buttons and marquee.

Reply:
Added a compressed version of curve-logo as well.